services/report/3rdparty/jasper-cs-report/src/main/resources/tombstone_with_budget.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<document name="report">
+  <ns2:reports_common xmlns:ns2="http://collectionspace.org/services/report">
+    <name>Tombstone with Budget</name>
+    <notes>Tombstone Report with Image and Budget (Public Art Profile)</notes>
+    <forDocTypes>
+      <forDocType>CollectionObject</forDocType>
+    </forDocTypes>
+    <supportsSingleDoc>false</supportsSingleDoc>
+    <supportsDocList>true</supportsDocList>
+    <supportsGroup>false</supportsGroup>
+    <supportsNoContext>true</supportsNoContext>
+    <filename>tombstone_with_budget.jrxml</filename>
+    <outputMIME>application/pdf</outputMIME>
```

Comment:
If this is intended to be a csv, you should set `outputMIME` to `text/csv`. That will make CSV the default format when the report is run.

Reply:
Yea I had been using pdf for testing so I could see the thumbnails but can switch things over to start using csv.

Reply:
That's a good point... I'm not sure what the intent is, if it's asking for an image in a csv-formatted report. We should probably clarify, but this looks good for now.